_includes/nav_list

```diff
@@ -9,11 +9,7 @@
       <li>
         {% if nav.url %}
           {% comment %} internal/external URL check {% endcomment %}
```

Comment:
Here's another one. Can remove the `{% comment %}` and `{% assign nav_url %}` and go with just `{{ nav-url | relative }}` instead of `{{ nav_url }}`.